src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +58,38 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetIsTopLevelWrapper(debugID, isTopLevelWrapper) {
```

Comment:
One problem I had is it’s hard to know whether something *is* a top level wrapper until it’s mounted and we can do the `instance._renderedComponent._currentElement === instance._currentElement.props` duck typing check.

I can add a special static field to its type so we can know to `_debugID` to `0` in `instantiateReactComponent` for it. Then `ReactDebugTool` can easily ignore any events about it as if they never happened.